src/Orchard.Specs/Media.feature

```diff
@@ -1,31 +1,13 @@
 ﻿Feature: Media management
     In order to reference images and such from content
     As an author
-    I want to upload and manage files in a media folder
+    I want to access the Media Library
 
 Scenario: Media admin is available
     Given I have installed Orchard
-        And I have installed "Orchard.Media"
+    And I have installed "Orchard.MediaLibrary"
 
-    # Accessing the media page
-    When I go to "admin/media"
-    Then I should see "Media"
-        And the status should be 200 "OK"
-
-    # Creating a folder
-    When I go to "admin/media/create"
-        And I fill in
-            | name | value |
-            | Name | Hello World |
-        And I hit "Save"
-        And I am redirected
-    Then I should see "Media"
-        And I should see "Hello World"
-        And the status should be 200 "OK"
-
-    # Editing a media with limited rights
-    When I go to "admin/media/edit?name=..\..\bin&mediaPath=..\..\bin"
-        And I am redirected
-    Then I should see "Media"
-        And I should see "Editing failed: Invalid path"
-        And the status should be 200 "OK"
+    # Accessing the Media Library page
+    When I go to "Admin/Orchard.MediaLibrary"
+    Then I should see "Media Library"
+    And the status should be 200 "OK"
```

Comment:
The Media module is deprecated, but I didn't really find an easy way to adapt these tests to Media Library, so I just adapter the basics and removed the rest.